docs/source/release_notes.rst

```diff
@@ -6,6 +6,43 @@
 Releases
 ========
 
+
+Version 1.3.0
+====================
+
+Notes
+-----
+
+* Update recommended
+
+  * No changes from version 1.3.0-beta.1 release. Collection promoted to certified content with version 1.3.0.
```

Comment:
My understanding is there were changes needed for sanity failures, correct? We can look through the merged PRs to see what else might have changed.